servers/visual/visual_server_scene.cpp

```diff
@@ -675,30 +704,238 @@ void VisualServerScene::instance_set_layer_mask(RID p_instance, uint32_t p_mask)
 
 	instance->layer_mask = p_mask;
 }
+
+void VisualServerScene::instance_teleport(RID p_instance) {
+	_interpolation_data.instance_teleport_list.push_back(p_instance);
+}
+
+void VisualServerScene::instance_set_interpolated(RID p_instance, bool p_interpolated) {
+	Instance *instance = instance_owner.get(p_instance);
+	ERR_FAIL_COND(!instance);
+	instance->interpolated = p_interpolated;
+}
+
 void VisualServerScene::instance_set_transform(RID p_instance, const Transform &p_transform) {
 	Instance *instance = instance_owner.get(p_instance);
 	ERR_FAIL_COND(!instance);
 
-	if (instance->transform == p_transform) {
-		return; //must be checked to avoid worst evil
-	}
+	if (!Engine::get_singleton()->is_physics_interpolation_enabled() || !instance->interpolated) {
```

Comment:
I commented the same in rocket chat, but would love to see any settings for interpolation not be engine/global.   As reduz suggested perhaps a property on scene tree, and perhaps something gets assigned when a node is entered in that tree.